tests/hooks/mod.rs

```diff
@@ -409,7 +409,7 @@ fn env_change_dont_panic_with_many_args() {
 
     let actual_repl = nu!(cwd: "tests/hooks", nu_repl_code(inp));
 
-    assert!(actual_repl.err.contains("IncompatibleParametersSingle"));
+    assert!(actual_repl.err.contains("incompatible_parameters"));
```

Comment:
Any idea why these tests needed to change? That's a bit surprising to me, I thought only the link would be removed by this PR.

Reply:
To me it seems that the ‘contains’ is checking for the error type, but the only place the pascal cased version of the error type exists in the string is in the doc url, so I changed it to be the snake cased version that still is present.